tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -1,7 +1,11 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
 exports[`access_hidden_namespace.test.w 1`] = `
-"error: Unknown symbol \\"core\\"
+"- Compiling invalid/access_hidden_namespace.test.w to tf-aws...
+
+✖ Compiling invalid/access_hidden_namespace.test.w to tf-aws...
+
+error: Unknown symbol \\"core\\"
```

Comment:
I wonder why these extra logs are ending up in our snapshots

Reply:
The regression you found was causing this. Just pushed up changes that fixed it.